bin/test_submit

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
```

Comment:
Is this file supposed to be committed to the repo?

Reply:
I guess I made this file at some point and forgot. I have removed it.